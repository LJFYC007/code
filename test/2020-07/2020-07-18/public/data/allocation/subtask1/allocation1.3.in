1
1 5
125744370 
605280149
225844028
1
249439392
410846048
