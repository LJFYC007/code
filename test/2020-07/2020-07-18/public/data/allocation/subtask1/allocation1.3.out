125744370.0
125744370.0
125744370.0
125744370.0
125744370.0
